README.md

```diff
@@ -1,53 +1,88 @@
-# Wiki
+# Publish to GitHub wiki action
 
-This action takes the contents of a provided directory and synchronizes it with your
-repository's wiki. This allows, for example, pull requests for the wiki.
+📖 Deploy docs from your source tree to the GitHub wiki
 
-The goal is to expose the wiki to PRs, so that potential contributors who aren't
-collaborators can contribute to the wiki and go through a review process. Alternatively,
-one can disabled the "Restrict editing to collaborators only" setting for their wiki,
-but this allows the community to make unreviewed, potentially unwanted, changes.
+## Features
 
-By default, it will use the `.github/wiki` directory.
+🌐 Works across repositories (with a [PAT]) \
+📚 Pretty interface for Markdown docs \
+⤴️ Lets you open PRs for wiki docs
 
-## Before using this action
+## Usage
+
+❗ Make sure you turn on the GitHub wiki feature in your repo's settings menu.
+You'll also need to _manually_ create a dummy page to initialize the git repo
+that powers the GitHub wiki. If you don't, when we push to `your-repo.wiki.git`,
+your workflow will fail because the wiki doesn't exist.
+
+<div align="center">
 
-You probably want to commit your wiki in its initial state before using this action.
-This is because this action will commit *all* differences between the wiki and the
-`.github/wiki` (or other specified directory) contents.
+![](https://user-images.githubusercontent.com/61068799/225441831-d3587ceb-0462-4591-bf00-ee56b040fe00.png)
```

Comment:
Don't see any harm in adding some alt text here.

Where did you find this link, BTW?

Reply:
Good idea on the alt text 👍

I dragged-and-dropped the image into the GitHub Web Editor. GitHub will store images for you that you drag-and-drop into Gists, GitHub files, comments, issues, PRs, pretty much anywhere there's a comment box. In this case, the image is that big hashed URL that is impossible to change. If you prefer https://imgur.com/upload, they also offer immutable images forever - that's what I use when I'm in VS Code instead of the GitHub Web Editor. Or, if you want it _in the repo_ I guess that's OK too; it's just not really _code_ so I hesitate to put markdown images into source control. 🤷‍♂️

Reply:
> Or, if you want it _in the repo_ I guess that's OK too; it's just not really _code_ so I hesitate to put markdown images into source control.

:thinking: Yeah, I think I'd want it in the repo TBH. As long as it's not a really high resolution and adds a lot of bytes to the repo 🙂